enterprise/internal/batches/service/service_test.go

```diff
@@ -2508,6 +2508,74 @@ changesetTemplate:
 			}
 		})
 	})
+
+	t.Run("UpsertEmptyBatchChange", func(t *testing.T) {
+		t.Run("creates new Batch Change if it is non-existent", func(t *testing.T) {
```

Comment:
nit: 

```suggestion
		t.Run("creates new batch change if it is non-existent", func(t *testing.T) {
```
product = upper case, entity = lower case